pyproject.toml

```diff
@@ -48,8 +48,6 @@ classifiers = [
   'Operating System :: POSIX :: Linux',
   'Programming Language :: C',
   'Programming Language :: Python',
-  'Programming Language :: Python :: 3.8',
-  'Programming Language :: Python :: 3.9',
```

Comment:
I'm unable to make it a GH code suggestion on deleted lines but I'd leave the "3.9" line as it was